wt-status.h

```diff
@@ -66,6 +82,8 @@ struct wt_status {
 	int show_branch;
 	int hints;
 
+	struct wt_status_vvp vvp;
```

Comment:
OK. That makes sense.  When I started this work, I wasn't sure how many fields I would end up needing.  And with the above simplifications, it's now just the SHA and verbose.